assets/js/opensuse-theme.js

```diff
@@ -431,3 +433,14 @@ window.fbAsyncInit = function() {
    fjs.parentNode.insertBefore(js, fjs);
  }(document, 'script', 'facebook-jssdk'));
 
+
+
+window.onkeyup = function(e) {
+   var key = e.keyCode ? e.keyCode : e.which;
+
+   if (key == 8) {
```

Comment:
Can you add the ESC key here as well? That would be really good and intuitive as well 👍